packages/@aws-cdk/aws-servicecatalog/lib/private/association-manager.ts

```diff
@@ -48,6 +49,34 @@ export class AssociationManager {
     }
   }
 
+  public static associateTagOptions(portfolio: IPortfolio, tagOptions: TagOptions): void {
+    Object.keys(tagOptions.tagOptionsMap).forEach(key => {
```

Comment:
You can use `Object.entries` with `for...of` instead, [like this](https://github.com/aws/aws-cdk/blob/ebfc0e092addbcf21562ce63328457285f7b2d77/packages/%40aws-cdk/aws-ec2/lib/cfn-init.ts#L291-L291), which will save you accessing the value separately with `tagOptions.tagOptionsMap[key]` below.